### PR TITLE
Update Aisleriot to 3.22.13

### PR DESCRIPTION
**Pull request by nyx-lyb3ra**
- [x] Runtime bumped to 3.38
- [x] Aisleriot updated to 3.22.13
- [x] Switched to meson build system
- [x] Updated dependencies
- [x] Added libcanberra 0.30

### CODE REVIEW

**Comment by nyx-lyb3ra** [comment]
bot, build

**Comment by nyx-lyb3ra** [comment]
Just noticed that there're newer tagged versions of Aisleriot on the GitLab repo. Should we be still using the ones available on download.gnome.org or can we switch?

CC @bilelmoussaoui

**Comment by bilelmoussaoui** [comment]
> Just noticed that there're newer tagged versions of Aisleriot on the GitLab repo. Should we be still using the ones available on download.gnome.org or can we switch?
> 
> CC @bilelmoussaoui

If the release is tagged I guess yes. Normally there should be a tarball, but Aisleriot is a special case.

**Comment by nyx-lyb3ra** [comment]
bot, build

File: org.gnome.Aisleriot.json [hunk 1]
```diff
@@ -79,14 +79,13 @@
         },
         {
             "name": "aisleriot",
-            "config-opts": [
-                "--with-card-theme-formats=svg"
-            ],
```
Comment (bilelmoussaoui) [inline org.gnome.Aisleriot.json]: Needs an equivalent from meson iirc

Comment (nyx-lyb3ra) [reply org.gnome.Aisleriot.json]: Well... checking https://gitlab.gnome.org/GNOME/aisleriot/-/blob/3.22.13/meson_options.txt it seems that is using svg-rsvg as default for `-Ddefault_theme_format`, and `-Dtheme_svg_rsvg` is true by default

**Comment by nyx-lyb3ra** [comment]
Great! We have an invalid AppData because the kudos were translated

**Comment by bilelmoussaoui** [comment]
> Great! We have an invalid AppData because the kudos were translated

aghh; not great :/

**Comment by nyx-lyb3ra** [comment]
@bilelmoussaoui Is there a way to get the files resulting from the build? I was thinking of creating a patch for the translations, thus eliminating the lines that should not be translated

**Comment by bilelmoussaoui** [comment]
> @bilelmoussaoui Is there a way to get the files resulting from the build? I was thinking of creating a patch for the translations, thus eliminating the lines that should not be translated

Sounds complex to patch the file after build? just remove the lines on the original metainfo file, if the lines are not present, they will not be translated

**Comment by nyx-lyb3ra** [comment]
@bilelmoussaoui I was thinking instead looking at the generated AppData to identify the languages ​​that translate things they shouldn't, and then patch the po files

Although removing kudos is a good idea too (although it also affects the URL that leads to Aisleriot's entry on the GNOME wiki), I just don't know if it's fair :/

**Comment by bilelmoussaoui** [comment]
> @bilelmoussaoui I was thinking instead looking at the generated AppData to identify the languages ​​that translate things they shouldn't, and then patch the po files
> 
> Although removing kudos is a good idea too (although it also affects the URL that leads to Aisleriot's entry on the GNOME wiki), I just don't know if it's fair :/

Just make sure to report that upstream. Though, it's very weird how gettext though those are translatable?? I would do a local build and check the generated appdata as well to see what really causes the issue.

**Comment by nyx-lyb3ra** [comment]
@bilelmoussaoui Well, I just found the source of the issue. It turns out that Aisleriot's AppData file had an xmlns set. Removing it, gettext works correctly again

**Comment by nyx-lyb3ra** [comment]
https://gitlab.gnome.org/GNOME/aisleriot/-/issues/45

**Comment by nyx-lyb3ra** [comment]
Hmm... seems to crash when the deck is clicked to draw a new card in Wayland. In X11 it works correctly

**Comment by nyx-lyb3ra** [comment]
Oh, known issue, we need libcanberra from git to fix that
https://gitlab.gnome.org/GNOME/aisleriot/-/issues/44

**Comment by nyx-lyb3ra** [comment]
Well, it does not longer crash on Wayland. It's still throwing some warnings, but at least you can play fine

**Comment by bilelmoussaoui** [comment]
thanks!

**Pull request merged**